.github/dependabot.yml

```diff
@@ -13,6 +13,7 @@ updates:
       libcnb:
         patterns:
           - "libcnb"
+          - "libcnb-test"
```

Comment:
(optional) Yeah I don't know what's up with Dependabot not grouping the dependencies properly when using the wildcard. Good idea to try listing the package names explicitly. Given all of the package names are now listed, the wildcard could be removed too?